pandas/core/frame.py

```diff
@@ -3649,7 +3649,9 @@ def reindexer(value):
     @property
     def _series(self):
         return {
-            item: Series(self._mgr.iget(idx), index=self.index, name=item)
+            item: Series(
+                self._mgr.iget(idx), index=self.index, name=item, fastpath=True
```

Comment:
we need the fastpath arg?

Reply:
in order to go through the fastpath, yes.  getting rid of the fastpath entirely is a separate topic

Reply:
kk